skins/laika/templates/Base_Layout.html.twig

```diff
@@ -28,7 +28,8 @@
 </head>
 <body>
 {% block main %}
-    <div id="app" data-application-vars="{$ _context|json_encode|e('html_attr') $}" 
+    <div id="app" />
+    <div id="appdata" data-application-vars="{$ _context|json_encode|e('html_attr') $}"
```

Comment:
This needs to be set to `display: none;` or the flex css rules will make it a column